### ISSUE TITLE
Select - can't set background color

### ISSUE THREAD

**Issue by giladaya** [jxnblk/rebass#458]
I tried this in the docs (https://jxnblk.com/rebass/components/Select):
```
<Select bg="red" css={{backgroundColor: 'blue'}}>
  <option>Hello</option>
  <option>Beep</option>
  <option>Boop</option>
</Select>
```
The background color doesn't change at all.

My workaround (in my code) was to define a styled component with a custom background color and use that:
```
const MySelect = styled(Select)`
    background-color: red;
`
```

**Comment by jxnblk** [comment]
In your example you have both a `bg` prop and `backgroundColor` defined. What is the expected behavior here?

**Comment by giladaya** [comment]
Actually I smooshed together two separate examples, both demonstrating that setting the background on a `Select` component doesn't work:

Example 1:
```
<Select bg="red">
  <option>Hello</option>
  <option>Beep</option>
</Select>
```
Expected behaviour: the background color of the `Select` component should be red

After Example 1 didn't work, I tried a different approach:
Example 2:
```
<Select css={{backgroundColor: 'blue'}}>
  <option>Hello</option>
  <option>Beep</option>
</Select>
```
Expected behaviour: the background color of the `Select` component should be blue

Example 2 didn't work either.

**Comment by jxnblk** [comment]
Thanks! Yeah that's definitely a bug – looks like this line should be `backgroundImage` not `background` https://github.com/jxnblk/rebass/blob/master/src/Select.js#L26

**Comment by jxnblk** [comment]
Should be fixed in `v2.3.0`
